### PR TITLE
Add bower.json file

### PR DESCRIPTION
**Pull request by tricinel**
Added bower.json file which defines the main file used. This will allow wire dependency tasks (grunt, gulp) to wire just the correct files needed for tota11y to run.

### CODE REVIEW

**Comment by tricinel** [comment]
[clabot:check]

**Comment by jdan** [comment]
Hey @tricinel. Thanks for taking the time to put this together. Did this come out of your own necessity to have a `bower.json` for a project? I ask because we made a conscious decision *not* to include a `bower.json` in tota11y because we didn't think it would be very useful.

**Comment by tricinel** [comment]
Hi @jdan, I like the ease of use when putting together a workflow with bower and gulp. With gulp, you can look through your bower_components (or where you're installing bower frontend components) and wire all the project's dependencies and inject them into your html file where needed.

For a single page app, when you want to do stuff fast, setting this up is painless and you don't have to do manual work when adding a new component. Just do bower install and gulp will inject the file(s) that new component uses. The catch is that the new component needs to have a bower.json file with the "main" property so that the gulp plugin will know which file to inject.

For me, it is useful, hopefully I'm not the only one :)

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "tota11y",
```
Comment (jdan) [inline bower.json]: Do we need a version number as well?

File: bower.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "authors": [
+    "Jordan Scales <scalesjordan@gmail.com>"
+  ],
+  "license": "MIT",
+  "keywords": [
+    "tota11y",
+    "accessibility",
+    "khan"
+  ],
+  "homepage": "https://github.com/Khan/tota11y",
```
Comment (jdan) [inline bower.json]: Maybe http://khan.github.io/tota11y would be better here

**Comment by jdan** [comment]
Awesome - thank you for the explanation :) I left a few comments inline, but I'm happy to merge this to make it easier to consume.

File: bower.json (hunk 1 above)
Comment (tricinel) [inline bower.json]: Indeed. We should add it, yes. For some reason, bower init didn't ask me for one, weird...

File: bower.json (hunk 2 above)
Comment (tricinel) [inline bower.json]: Agreed.

**Comment by tricinel** [comment]
Awesome. Let me know if you want me to make those changes to bower.json (version number and homepage change).

Cheers!

**Comment by jdan** [comment]
@tricinel Yes if you don't mind! I can merge right after. Thanks again.

**Comment by tricinel** [comment]
Done. No problem, thank you for the tool :)

**Comment by jdan** [comment]
Woohoo! :star2:

**Comment by tricinel** [comment]
Awesome! Back to building stuff :+1: 

Thanks.

**Pull request merged**